README.md

```diff
@@ -1 +1,59 @@
-# Week-One-Capstone
\ No newline at end of file
+# ![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone
+![screenshot](Capture.JPG)
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Live Demo
+
+[Github Pages Link](https://fuadnabi.github.io/Week-One-Capstone/)
+
+
+### Setup
+If you want to try this application locally, follow the steps below:
+
+1. Clone the project using git clone `git@github.com:FuadNabi/Week-One-Capstone.git`.
+
+2. Run `cd Week-One-Capstone`.
+
+3. Run `npm install`.
+
+4. Run `npm start`.
```

Comment:
- [ ] Since you did not use the webpack, kindly consider removing these two commands for they are not applicable to this project.
`Hint: You may consider telling how the project may be run locally by opening it up with the liveserver.`

Reply:
Corrected!